system/lib/wasmfs/wasmfs.cpp

```diff
@@ -0,0 +1,55 @@
+// // Copyright 2021 The Emscripten Authors.  All rights reserved.
+// // Emscripten is available under two separate licenses, the MIT license and the
+// // University of Illinois/NCSA Open Source License.  Both these licenses can be
+// // found in the LICENSE file.
+// // wasmfs.cpp will implement a new file system that replaces the existing JS filesystem.
+// // Current Status: Work in Progress.
+// // See https://github.com/emscripten-core/emscripten/issues/15041.
+
+#include <stdlib.h>
+#define __NEED_struct_iovec
```

Comment:
what does this do?

Reply:
Oh, I can remove that, it was from a prior definition of `iovec`. Realized that we already have it defined in `api.h`. Thanks